lib/utilities.js

```diff
@@ -6,24 +6,19 @@
  * @param  {String} string - The string.
  * @return {String}
  */
+
+var _hyphenPattern = /-(.)/g;
```

Comment:
Would you mind moving this line above the JSDoc? This is because we can find easier to read the comment when it's next to the function.